doc/related_projects.rst

```diff
@@ -150,6 +159,17 @@ and tasks.
 - `multiisotonic <https://github.com/alexfields/multiisotonic>`_ Isotonic
   regression on multidimensional features.
 
+- `libOPF <https://github.com/LibOPF/LibOPF>`_ Optimal path forest classifier
+
+- `random-output-trees <https://github.com/arjoly/random-output-trees>`_
+  Randomized output tree for multilabel / multi-output regression tasks
+
+- `fastFM <https://github.com/ibayer/fastFM>`_ Fast factorization machine
```

Comment:
"Factorization" makes me pretty sure this belongs with decomposition, not classification/regression